{
  "_annotation": {
    "cve_id": "CVE-2024-21513",
    "generated_from": "https://raw.githubusercontent.com/anchore/cve-data-enrichment/main/data/anchore/2024/CVE-2024-21513.json",
    "reason": "Added CPE configurations because not yet analyzed by NVD."
  },
  "cve": {
    "configurations": [
      {
        "nodes": [
          {
            "cpeMatch": [
              {
                "criteria": "cpe:2.3:a:langchain:langchain-experimental:*:*:*:*:*:python:*:*",
                "matchCriteriaId": "1A7CFE26-82A6-537F-AF36-A082B2856499",
                "versionEndExcluding": "0.0.21",
                "versionStartIncluding": "0.0.15",
                "vulnerable": true
              }
            ],
            "negate": false,
            "operator": "OR"
          },
          {
            "cpeMatch": [
              {
                "criteria": "cpe:2.3:a:langchain:langchain_experimental:*:*:*:*:*:python:*:*",
                "matchCriteriaId": "93AB85E1-3769-57DC-8CDA-EB75FBB22A18",
                "versionEndExcluding": "0.0.21",
                "versionStartIncluding": "0.0.15",
                "vulnerable": true
              }
            ],
            "negate": false,
            "operator": "OR"
          }
        ]
      }
    ]
  }
}